package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "vui-typography",
-  "version": "0.6.0",
+  "version": "0.7.0",
```

Comment:
Maybe we want to go with `1.0.0` for the C12 versions? That will leave room to make further changes in the next year to the `0.x` line.